packages/backend/src/core/reports/getTotalSupplyFormulaConfigHash.ts

```diff
@@ -1,19 +1,18 @@
 import { hashJson, Token } from '@l2beat/shared-pure'
 
-const EBV_LOGIC_VERSION = 0
+const LOGIC_VERSION = 0
 
-export function getEBVConfigHash(tokens: Token[]) {
-  return hashJson([extractEBVTokens(tokens), EBV_LOGIC_VERSION])
+export function getTotalSupplyFormulaConfigHash(tokens: Token[]) {
+  return hashJson([extractTokens(tokens), LOGIC_VERSION])
 }
 
-function extractEBVTokens(tokens: Token[]) {
+function extractTokens(tokens: Token[]) {
```

Comment:
maybe we should add an assert here to be sure that there are only totalSupply tokens?